src/main/java/org/fundacionjala/coding/christian/Kata15.java

```diff
@@ -0,0 +1,25 @@
+package org.fundacionjala.coding.christian;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Developer by Christian.
+ */
+public class Kata15 {
+    /**
+     * @param collection .
+     * @return int
+     */
+    public int mostFrequentItemCount(int[] collection) {
+        Map<Integer, Integer> counter = new HashMap<Integer, Integer>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)